libmamba/src/core/context.cpp

```diff
@@ -177,7 +178,7 @@ namespace mamba
     Context::Context(const ContextOptions& options)
     {
         on_ci = static_cast<bool>(util::get_env("CI"));
-        prefix_params.root_prefix = util::get_env("MAMBA_ROOT_PREFIX").value_or("");
+        prefix_params.root_prefix = detail::get_root_prefix();
```

Comment:
That's the only part I'm not sure about, since `prefix_params.root_prefix` is used to store the root_prefix set by the config (see https://github.com/mamba-org/mamba/blob/main/libmamba/src/api/configuration.cpp#L1237-L1245). I don't know whether some parts of the code do something specific when this variable is empty, nor if there are use cases where it would be valid to have it empty (when using the library out of mamba / micromamba for instance).